src/pygeoprocessing/symbolic.py

```diff
@@ -170,10 +171,11 @@ def _generic_raster_op(*arg_list):
     target_dtype = numpy.result_type(*[x.dtype for x in array_list])
     result = numpy.empty(arg_list[0].shape, dtype=target_dtype)
     nodata_list = arg_list[n:2*n]
-    func = arg_list[2*n]
+    expression = arg_list[2*n]
     target_nodata = arg_list[2*n+1]
     default_nan = arg_list[2*n+2]
     default_inf = arg_list[2*n+3]
+    kwarg_names = arg_list[-1]

```

Comment:
Can I request the following to be consistent with the rest of the nodation of the indexes, the docstring, and avoiding a nasty future bug caused by a change in the length of the list?

```suggestion
    kwarg_names = arg_list[2*n+4]
```